data/json/items/chemicals_and_resources.json

```diff
@@ -1534,5 +1546,13 @@
     "container": "test_tube",
     "symbol": ";",
     "color": "light_green"
+  },
+  {
+    "id": "conc_paralytic",
+    "type": "AMMO",
+    "name": { "str_sp": "concentrated paralytic" },
+    "description": "The concentrated paralytic agent, originally from a triffid.  Given your experience with its effects (chiefly gathered while sourcing this batch) and their striking similarity to earthly flora you figure it's related to curarines.  Might come in handy in crafting more useful things.",
```

Comment:
Comma between "flora" and "you" might make this description flow better

Reply:
Thanks, commas are fiddly